scripts/bump.sh

```diff
@@ -7,12 +7,12 @@ if [ "$#" -lt 1 ]; then
   exit 1
 fi
 
-if [ "$(git rev-parse --abbrev-ref HEAD)" != "master" ]; then
+if [ "$(git rev-parse --abbrev-ref HEAD)" != "stable" ]; then
```

Comment:
I am partly tempted to remove this file completely to avoid confusing people, but I left it intact just in case there are reasons to version Vega-lite manually outside of what is being calculated using semantic commit messages.

Reply:
I'm okay with removing it. I made the file because I wanted a way to bump the version and update all the required files without having to remember the commands. I don't think we need it anymore with your process.

Reply:
Ok! If we need it again, it can be retrieved from git history. Removed in https://github.com/vega/vega-lite/pull/7984/commits/334cfeb7f806331b979691cf74e85e57acd31360